File: Estrutura_Sequencial/Java/Ex1019/Main.java
package Ex1019;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int segundos = sc.nextInt();

        int horas = segundos / 3600;
        segundos %= 3600;
        int minutos = segundos / 60;
        segundos %= 60;

        System.out.printf("%d:%d:%d\n", horas, minutos, segundos);

        sc.close();
    }
}

File: Estrutura_Sequencial/C/Ex1019/main.c
#include <stdio.h>

int main() {
    int segundos;
    scanf("%d", &segundos);

    int horas = segundos / 3600;
    segundos %= 3600;

    int minutos = segundos / 60;
    segundos %= 60;

    printf("%d:%d:%d\n", horas, minutos, segundos);

    return 0;
}
